packages/core/src/App/Containers/WelcomeModal/welcome-item.jsx

```diff
@@ -70,7 +70,7 @@ WelcomeItem.propTypes = {
     description: PropTypes.element.isRequired,
     icon: PropTypes.element.isRequired,
     mobileIcon: PropTypes.element.isRequired,
-    onClick: PropTypes.func.isRequired,
+    onClick: PropTypes.func,
```

Comment:
Not-sure section is not clickable now. Thus, OnClick function is not a required prop